ci/lint.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.17.1
```

Comment:
I'd make the linter use `latest`, only builds need to be pinned to a specific version. (Similar to audit.yml)